CMakeLists.txt

```diff
@@ -77,6 +77,16 @@ if(BUILD_TESTING)
     target_link_libraries(test_concat c_utilities)
   endif()
 
+  ament_add_gtest(test_string_array
+    test/test_string_array.cpp
+  )
+  if(TARGET test_string_array)
+    target_include_directories(test_string_array PUBLIC
+      ${c_utilities_INCLUDE_DIRS}
```

Comment:
Can you please double check the content of this variable. I doubt that it contains anything and also since `include_directories` is called before shouldn't be necessary.